package.json

```diff
@@ -10,11 +10,11 @@
   "scripts": {
     "_rushInstall": "node common/scripts/install-run-rush.js install --purge",
     "_rushBuild": "node common/scripts/install-run-rush.js build",
-    "_rushBuildToExperiments": "node common/scripts/install-run-rush.js build --to @uifabric/experiments",
+    "_rushBuildToFabricResources": "node common/scripts/install-run-rush.js build --to @uifabric/fabric-website-resources",
```

Comment:
Is fabric-website-resources before or after experiments?

Reply:
I think it is before, I think the correct thing would be to use `fabric-website` here.

Reply:
This is before. I don't believe experiments is required in order to run `npm start`, and would prefer not to add unnecessary steps as this is already a long process.

Reply:
Someone wanted to build to experiments and pretty recently made the change to do that. You should look at the change history and talk to whoever it was before reverting.